src/discolight/augmentations/randomscale.py

```diff
@@ -0,0 +1,40 @@
+"""An augmentation to randomly rotate an image."""
+import random
+import math
+from discolight.params.params import Params
+from .augmentation.types import Augmentation, NumericalRange
+from .scale import Scale
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class RandomScale(Augmentation):
+
+    """Randomly scale the given image."""
+
+    def __init__(self, scale_range):
+        """Construct a RandomScale augmentation.
+
+        You should probably use the augmentation factory or Discolight
+        library interface to construct augmentations. Only invoke
+        this constructor directly if you know what you are doing.
+        """
+        super().__init__()
+
+        self.scale_range = scale_range
+
+    @staticmethod
+    def params():
+        """Return a Params object describing constructor parameters."""
+        return Params().add(
+            "scale_range",
+            "The scale range should be bigger than -1",
+            NumericalRange(-1, math.inf), (0.2, 0.2))

```

Comment:
```suggestion
            NumericalRange(-1), (0.2, 0.2))

```

In general, you should use `None` instead of `±math.inf`. Note that if there is no maximum, you can simply leave it out, like I've done here in the suggestion.